src/angular/pagination/paginator/paginator.scss

```diff
@@ -1,63 +1,80 @@
 @import '../pagination';
 
+$paginatorSize: 40;
+$paginatorSize4K: $paginatorSize * $scalingFactor4k;
+$paginatorSize5K: $paginatorSize * $scalingFactor5k;
+$paginatorSizeLean: 30;
+
+$paginatorDisabledColor: $sbbColorSilver;
+$paginatorDisabledColorLean: $sbbColorStorm;
+
 .sbb-paginator {
   display: flex;
   padding-left: 0;
   list-style: none;
-  margin-left: 0;
+  margin: 0;
```

Comment:
Is there a need to manually set the margin to 0? Do we inherit it from somewhere?

Reply:
Yes from ul / li styles. I will add a comment do clarify